target_athena/athena.py

```diff
@@ -40,8 +50,210 @@ def create_client(config, logger: Logger):
         ).cursor()
     return cursor
 
-
 def execute_sql(sql, athena_client):
+    """Run sql expression using athena client
+
+    Args:
+        sql (string): a valid sql statement string
+        athena_client ([type]): [description]
+    """
     athena_client.execute(sql)
-    # print(cursor.description)
-    # print(cursor.fetchall())
\ No newline at end of file
+
+def table_exists(athena_client, database, table_name):
+    """Determine if a table already exists in athena.
+
+    Args:
+        athena_client ([type]): [description]
+        database ([type]): [description]
+        table_name ([type]): [description]
```

Comment:
We can come back and add these types and descriptions, but just wanted to call them out here in case these were intended to be already populated.

Reply:
Yeah, I just wanted to at least generate a skeleton of docstrings for the time being.  I figure we'll do some refactoring once core functionality is in place.